pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -354,6 +354,10 @@ func (c *Reconciler) reconcileDeployment(el *v1alpha1.EventListener) error {
 				existingDeployment.Spec.Template.Spec.Containers[0].Command = nil
 				updated = true
 			}
+			if len(existingDeployment.Spec.Template.Spec.Containers[0].VolumeMounts) == 0 {
+				existingDeployment.Spec.Template.Spec.Containers[0].VolumeMounts = container.VolumeMounts
+				updated = true
+			}
```

Comment:
Do we also need to check if the `existingDeployment.Spec.Template.Spec.Volumes` has been updated?

Reply:
I think we should add this below line 360:
```go
if !reflect.DeepEqual(existingDeployment.Spec.Template.Spec.Volumes, deployment.Spec.Template.Spec.Volumes) {
	existingDeployment.Spec.Template.Spec.Volumes =  deployment.Spec.Template.Spec.Volumes
	updated = true
}
```

Reply:
Added VolumeMount but not the actual Volume so the deployment is invalid :face_with_head_bandage: